### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
**What type of PR is this?**

/kind cleanup
/kind rule-update

**Any specific area of the project related to this PR?**

/area build
/area engine
/area rules
/area tests
/area proposals

**What this PR does / why we need it**:

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/falco/commit/fe4788d51b99f7dae4213e57921cd3dbef37adae#commitcomment-67427362

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/falco/commit/7978e6869a54a620ceb941e04b3af90e17042c2f

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

**Does this PR introduce a user-facing change?**:

```release-note
rule(Detect outbound connections to common miner pool ports): fix url in rule output
rule(macro somebody_becoming_themself): renaming macro to somebody_becoming_themselves
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1042,7 +1042,7 @@ Released 2019-06-07
 
 * Add instructions for k8s audit support in >= 1.13 [[#608](https://github.com/falcosecurity/falco/pull/608)]
 
-* Fix security issues reported by GitHub on Anchore integration [[#592](https://github.com/falcosecurity/falco/pull/592)]
+* Fix security issues reported by GitHub on Anchor integration [[#592](https://github.com/falcosecurity/falco/pull/592)]
```
Comment (jsoref) [inline CHANGELOG.md:1045]: this is wrong, turns out it's a brand

File: CHANGELOG.md [hunk 2]
```diff
@@ -1121,7 +1121,7 @@ Released 2019-05-13
 
 ### Rule Changes
 
-* **Tag rules using Mitre Attack Framework**: Add tags for all relevant rules linking them to the [MITRE Attack Framework](https://attack.mitre.org). We have an associated [blog post](https://sysdig.com/blog/mitre-attck-framework-for-container-runtime-security-with-sysdig-falco/). [[#575](https://github.com/falcosecurity/falco/pull/575)] [[#578](https://github.com/falcosecurity/falco/pull/578)]
+* **Tag rules using Mitre Attack Framework**: Add tags for all relevant rules linking them to the [MITRE Attack Framework](https://attack.mitre.org). We have an associated [blog post](https://sysdig.com/blog/mitre-attack-framework-for-container-runtime-security-with-sysdig-falco/). [[#575](https://github.com/falcosecurity/falco/pull/575)] [[#578](https://github.com/falcosecurity/falco/pull/578)]
```
Comment (jsoref) [inline CHANGELOG.md:1124]: will drop

File: proposals/20200901-artifacts-cleanup.md [hunk 3]
```diff
@@ -90,7 +90,7 @@ This way, assuming the number of prebuilt drivers does not skyrocket, we can rea
 
 Notice that, in case a Falco stable release will not depend on a new driver version, this means the last two driver versions will, in this case, cover more than the two Falco stable releases.
 
-### Archivation
+### Archiving
```
Comment (jsoref) [inline proposals/20200901-artifacts-cleanup.md:93]: ?

File: rules/falco_rules.yaml [hunk 4]
```diff
@@ -2193,7 +2193,7 @@
 #   output: "sshd sent error message to syslog (error=%evt.buffer)"
 #   priority: WARNING
 
-- macro: somebody_becoming_themself
+- macro: somebody_becoming_themselves
```
Comment (jsoref) [inline rules/falco_rules.yaml:2196]: happy to drop if people have a strong opinion

File: rules/falco_rules.yaml [hunk 5]
```diff
@@ -2818,7 +2818,7 @@
   desc: Miners typically connect to miner pools on common ports.
   condition: net_miner_pool and not trusted_images_query_miner_domain_dns
   enabled: false
-  output: Outbound connection to IP/Port flagged by cryptoioc.ch (command=%proc.cmdline port=%fd.rport ip=%fd.rip container=%container.info image=%container.image.repository)
+  output: Outbound connection to IP/Port flagged by https://cryptoioc.ch (command=%proc.cmdline port=%fd.rport ip=%fd.rip container=%container.info image=%container.image.repository)
```
Comment (jsoref) [inline rules/falco_rules.yaml:2821]: Fwiw, this site is dead.

**Review by jasondellaluce — APPROVED** [review]
/approve

Thank you @jsoref, this is such a great contribution! Would you consider running this on https://github.com/falcosecurity/libs and on https://github.com/falcosecurity/falco-website as well?

**Review by leogr — APPROVED** [review]
/approve

**Comment by jsoref** [comment]
Sure, added to my todo list https://github.com/stars/jsoref/lists/to-check-spelling (which is quite long).

**Pull request merged**